tests/basic/pure_test.go

```diff
@@ -5109,6 +5110,44 @@ var _ = Describe("{ValidatePodNameinVolume}", func() {
 				}
 			}
 
+		})
+		stepLog = "Validate the volume name in the FA"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			storageNodes := node.GetStorageNodes()
+			opts := node.ConnectionOpts{
+				IgnoreError:     false,
+				TimeBeforeRetry: defaultRetryInterval,
+				Timeout:         defaultTimeout,
+				Sudo:            true,
+			}
+			cmd := "cat /etc/pwx/cluster_uuid"
+			clusterUUID, err := Inst().N.RunCommand(storageNodes[0], cmd, opts)
+			log.FailOnError(err, "Failed to get cluster UUID")
+			log.InfoD("Cluster UUID [%v]", clusterUUID)
+			parts := strings.Split(clusterUUID, "-")
+			clusterUUIDfirstPart := parts[0]
+			GetVolumeNameFromPvc := func(namespace string, pvclist []string) []string {
+				allPvcList, err := core.Instance().GetPersistentVolumeClaims(namespace, nil)
+				log.FailOnError(err, fmt.Sprintf("error getting pvcs from namespace [%s]", namespace))
+				for _, p := range allPvcList.Items {
+					pvclist = append(pvclist, p.Spec.VolumeName)
+				}
+				return pvclist
+			}
+			listofPvcNames := make([]string, 0)
+			for _, ctx := range contexts {
+				listofPvcNames = GetVolumeNameFromPvc(ctx.App.NameSpace, listofPvcNames)
+			}
+			for _, pvcName := range listofPvcNames {
+				expectedVolName := PodNameinFA + "::" + "px_" + clusterUUIDfirstPart + "-" + pvcName
```

Comment:
if realm exists, we need to change this

Reply:
in this testcase we have realm , actually podNameinFA consists of realm name too

Reply:
correct the variable name, maybe confusing

Reply:
changed it